### PR TITLE
#1871 - Added Shanghai md file

### PR DESCRIPTION
**Pull request by rashmisubhash**

### CODE REVIEW

**Review by jbampton — CHANGES_REQUESTED** [review]
Both files and folders need to be lowercase

File: doc/source/shanghai.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Shanghai
```
Comment (jbampton) [inline doc/source/shanghai.md:1]:
```suggestion
# 🇨🇳 Shanghai
```

File: doc/source/shanghai.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# CN Shanghai
```
Comment (jbampton) [inline doc/source/shanghai.md:1]:
```suggestion
# 🇨🇳 Shanghai
```

**Comment by jbampton** [comment]
Both files and folders need to be lowercase 

Screen Shot 2022-10-01 at 4 15 04 pm

**Review by jbampton — CHANGES_REQUESTED** [review]
Both files and folders need to be lowercase

You need to fix the rest of the paths in the Markdown.

Also you need to rename the files and folders so they are lowercase

File: doc/source/shanghai.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+## Longhua Temple and Pagoda
+
+Longhua Temple is one of the oldest religious sites in China and stands next to a 40m-high brick and wood pagoda that was built in 242 AD. It consists of five large rooms where you can find a great statue of Buddha, the Great Hall of the Great Sage and the Hall of the Heavenly King. Among the highlights is the 5-ton, 2-meter-high bell tower built in 1382 and the impressive view of approximately 500 Buddhas.
+
+## The Oriental Pearl Tower
+
+This tower is 468 meters high and offers you an excellent view over the historic Bund Promenade, its name is due to 11 spheres of various sizes, the tallest of them being the Spatial Module where the outer shell and the floor are of glass. The tower offers you 15 observation areas, including a revolving restaurant with an excellent panoramic view.
+
+![The Oriental Pearl Tower](_static/images/shanghai/the-oriental-pearl-tower.jpg)
+
```
Comment (jbampton) [inline doc/source/shanghai.md:34]:
```suggestion
```

File: doc/source/shanghai.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+It is the largest gallery in Asia and houses the most important works of modern art in China and if you are an art lover in general you should also visit the Center for Oriental Art, where you can watch classical music, opera, ballet and theater performances.
+
+## The Maglev
+
+The Maglev is a train line that operates at high speed by means of magnetic levitation, it is the fastest transport service in the world and reaches a speed of up to 431 km/h, traveling the 30-kilometer stretch that goes from the Pudong International Airport to Longyand Road Station in about 7 minutes.
+
+![The Maglev](_static/images/shanghai/the-maglev.jpg)
+
+Shanghai, a modern city full of interesting places to visit!
```
Comment (jbampton) [inline doc/source/shanghai.md:68]:
```suggestion
Shanghai, a modern city full of interesting places to visit!

## About the Author

Idais, Graduated in Mechanical Engineering, and a master’s degree in teaching component, she gave classes in several institutes of mathematics and physics, but she also dedicated several years of my life as a television producer, she did the scripts for mikes, the camera direction, editing of video and even the location. Later she was dedicated to SEO writing for a couple of years. She likes poetry, chess and dominoes.
```

**Review by jbampton — CHANGES_REQUESTED** [review]
Broken links

https://github.com/slurpcode/slurp/actions/runs/3163610827/jobs/5151306327

Files and folders are still uppercase

https://github.com/slurpcode/slurp/pull/1875/files

**Comment by rashmisubhash** [comment]
> Broken links
> 
> https://github.com/slurpcode/slurp/actions/runs/3163610827/jobs/5151306327
> 
> Files and folders are still uppercase
> 
> https://github.com/slurpcode/slurp/pull/1875/files

Actually they aren't, but in the changes it is not showing

**Comment by jbampton** [comment]
This is sort of a known issue due to the fact that NTFS is case-insensitive.

Basically a problem on Windows

Try the solution at the link.

https://social.technet.microsoft.com/Forums/Lync/en-US/075300e7-64c8-4520-bee2-65ce435a4e51/changing-the-capitalization-of-a-filename-doesnt-work?forum=w7itprogeneral

**Review by jbampton — APPROVED** [review]

**Pull request merged**